src/app/utils/sample-url-generation.ts

```diff
@@ -10,10 +10,17 @@ export function parseSampleUrl(url: string, version?: string) {
     const urlObject: URL = new URL(url);
     requestUrl = urlObject.pathname.substr(6).replace(/\/$/, '');
     queryVersion = (version) ? version : urlObject.pathname.substring(1, 5);
-    search = decodeURI(urlObject.search);
+
+    const searchParameters = urlObject.search;
+    if (searchParameters && searchParameters !== '') {
```

Comment:
The condition will be fine without the second check  `searchParameters !== ''` . An empty string evaluates to a falsey value.